packages/flux-vision/src/index.ts

```diff
@@ -0,0 +1,63 @@
+// add html elements we need to pull data off of
```

Comment:
lets wrap this all in a nice big `install` function:

```
export default function() {
  // ...
}
```

then, when using it they wil just have to call `FluxVision.install()`